client/settings/deposits/index.js

```diff
@@ -0,0 +1,230 @@
+	const [
+		depositScheduleWeeklyAnchor,
+		setDepositScheduleWeeklyAnchor,
+	] = useDepositScheduleWeeklyAnchor();
+	const [
+		depositScheduleMonthlyAnchor,
+		setDepositScheduleMonthlyAnchor,
+	] = useDepositScheduleMonthlyAnchor();
+
+	const handleIntervalChange = ( newInterval ) => {
+		switch ( newInterval ) {
+			case 'weekly':
+				setDepositScheduleWeeklyAnchor(
+					depositScheduleWeeklyAnchor || 'monday'
+				);
+				break;
+
+			case 'monthly':
+				setDepositScheduleMonthlyAnchor(
+					depositScheduleMonthlyAnchor || '1'
+				);
+				break;
+		}
+
+		setDepositScheduleInterval( newInterval );
+	};
+
+	return (
+		<>
+			<div className="schedule-controls">
+				<SelectControl
+					label={ __( 'Frequency', 'woocommerce-payments' ) }
+					value={ depositScheduleInterval }
+					onChange={ handleIntervalChange }
+					options={ [
+						{
+							value: 'daily',
+							label: __( 'Daily', 'woocommerce-payments' ),
+						},
+						{
+							value: 'weekly',
+							label: __( 'Weekly', 'woocommerce-payments' ),
+						},
+						{
+							value: 'monthly',
+							label: __( 'Monthly', 'woocommerce-payments' ),
+						},
+					] }
+				/>
+				{ 'monthly' === depositScheduleInterval && (
+					<SelectControl
+						label={ __( 'Date', 'woocommerce-payments' ) }
+						value={ depositScheduleMonthlyAnchor }
+						onChange={ setDepositScheduleMonthlyAnchor }
+						options={ monthlyAnchors }
+					/>
+				) }
+				{ 'weekly' === depositScheduleInterval && (
+					<SelectControl
+						label={ __( 'Day', 'woocommerce-payments' ) }
+						value={ depositScheduleWeeklyAnchor }
+						onChange={ setDepositScheduleWeeklyAnchor }
+						options={ daysOfWeek }
+					/>
+				) }
+			</div>
+			<p className="help-text">
+				{ 'monthly' === depositScheduleInterval &&
+					__(
+						'Deposits scheduled on a weekend will be sent on the next business day.',
+						'woocommerce-payments'
+					) }
+				{ 'weekly' === depositScheduleInterval &&
+					__(
+						'Deposits that fall on a holiday will initiate on the next business day.',
+						'woocommerce-payments'
+					) }
+				{ 'daily' === depositScheduleInterval &&
+					__(
+						'Deposits will occur every business day.',
+						'woocommerce-payments'
+					) }
+			</p>
+		</>
+	);
+};
+const DepositsSchedule = () => {
+	const depositStatus = useDepositStatus();
+	const completedWaitingPeriod = useCompletedWaitingPeriod();
+
+	if ( 'enabled' !== depositStatus ) {
+		return (
+			<Notice status="warning" isDismissible={ false }>
+				<span>
+					{ __(
+						'Deposit scheduling is currently unavailable for your store.',
+						'woocommerce-payments'
+					) }
+				</span>
+			</Notice>
+		);
+	}
+	if ( true !== completedWaitingPeriod ) {
+		return (
+			<Notice status="warning" isDismissible={ false }>
+				<span>
+					{ __(
+						'Your first deposit will be held for 7 days. Deposit scheduling will be available after this period.',
+						'woocommerce-payments'
+					) }
```

Comment:
From https://github.com/Automattic/woocommerce-payments/issues/4464:
```
The "7" above is derived from deposits_delay_days
```

```suggestion
					{ sprintf(
						__(
							'Your first deposit will be held for %s days. Deposit scheduling will be available after this period.',
							'woocommerce-payments'
						),
						depositDelayDays
					) }
```

Reply:
I thought the same thing originally but was corrected. The first deposit is always held for 7 days as an anti-fraud measure.
The `delay_days` is the delay between when a payment is received and when it is available for deposits and can vary by country.

see p1661909116777449/1661907696.353099-slack-C03KT8NLL13